src/frontends/common/include/openvino/frontend/manager.hpp

```diff
@@ -59,6 +59,9 @@ class FRONTEND_API FrontEndManager final {
     FrontEnd::Ptr load_by_model(const Types&... vars) {
         return load_by_model_impl({ov::Any{vars}...});
     }
+    FrontEnd::Ptr load_by_model(const std::vector<ov::Any>& variants) {
+        return load_by_model_impl(variants);
+    }
```

Comment:
not sure that we need this change because `GraphIterator` should be automatically wrapped in `ov::Any`.

Reply:
It is not wrapped to but `ov::Any` as `GraphIterator` is not included in `Types`, so the change is needed.

I made this similar to frontend.load(), because currently such logic works only for frontend.load().
https://github.com/openvinotoolkit/openvino/blob/ee659c1ce8a70850e1490629ef56fcd48d716585/src/frontends/common/include/openvino/frontend/frontend.hpp#L68

Reply:
Hmm, OVTF recently passed `GraphIterator` to `frontend.load` as you can see in the implementation of `load` there is unwrapping logic for `GraphIterator`. Really it looks redundant